test.js

```diff
@@ -1,57 +1,56 @@
+import {Writable as WritableStream} from 'stream';
 import test from 'ava';
-import hookStd from 'hook-std';
 import Ora from './';
 
 const spinnerChar = process.platform === 'win32' ? '-' : '⠋';
 
-function readOutput(spinner, callback) {
-	let out = '';
-
-	const unhook = hookStd.stderr({silent: true}, output => {
-		out += output;
+let stream;
+let output;
+
+test.beforeEach(() => {
+	const noop = () => {};
+	output = '';
+	stream = new WritableStream({
```

Comment:
Can you use [`get-stream`](https://github.com/sindresorhus/get-stream) here instead?